packages/live-share-inking/src/canvas/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the Microsoft Live Share SDK License.
+ */
+
+export * from "./InkingCanvas";
+export * from "./DryWetCanvas";
```

Comment:
minor nit but I try to order the re-exports in an index.ts alphabetically. That's just so its easier to see if you're missing an export in a larger file since you won't get a compiler error.

Reply:
Funny - I _try_ to order them by inheritance. But sure, I can change to alphabetical order.

Reply:
Fixed in https://github.com/microsoft/live-share-sdk/commit/18b13a631f0c2833a77b7d5181cb89b36206c512